includes/class-multisite-directory-taxonomy.php

```diff
@@ -133,4 +135,32 @@ public function saveTermGeo ($term_id) {
         }
     }
 
+    /**
+     * Adds a dropdown box to filter directory entries by category in the admin listing
+     *
+     * @link https://developer.wordpress.org/reference/hooks/restrict_manage_posts/
+     *
+     * @param string $post_type
+     */
+    public static function add_taxonomy_admin_filter ($post_type) {
+
+        if (Multisite_Directory_Entry::name === $post_type) {
+            $tax_obj = get_taxonomy(self::name);
+            $tax_name = $tax_obj->labels->name;
+            $terms = get_terms(self::name);
+            if (count($terms) > 0) {
+                echo '<select name="' . esc_attr(self::name) . '" id="' . esc_attr(self::name) . '" class="postform">';
```

Comment:
Uh.. this came straight from the original snippet I've found online ^^' There doesn't seem to be any other reference to it in the original WP code, nor there is a style defined for it, so I'd say it's pretty safe to remove it. May I update this in a new commit?